pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractTopic.java

```diff
@@ -608,6 +609,10 @@ public String getReplicatorPrefix() {
     }
 
     private boolean allowAutoUpdateSchema() {
+        if (topic.contains(EventsTopicNames.TRANSACTION_BUFFER_SNAPSHOT)
```

Comment:
It seems that you can use the static method `EventsTopicNames#checkTopicIsEventsNames`.

Reply:
We should allow auto-upload schema for all system topics? Otherwise, for pulsar internal system topic, users need to upload the schema or we need to provide a way to generate schema, looks confused for users.

Reply:
BrokerService#isSystemTopic can help.

Reply:
Great suggestion, I also think we should allow auto-upload schema for all system topics.
But now the problem is that this change makes some tests unstable, I mean these tests pass when run alone, but give errors when run with other tests

Reply:
Solved, the reason is cleanup, the configuration modified in the test cannot be cleaned up